### PR TITLE
Add Java / WebGL bindings for screen-space reflections

### PR DESCRIPTION
**Pull request by bejado**

### CODE REVIEW

**Comment by prideout** [comment]
🍾  Sweet I will take a look!

**Review by pixelflinger — APPROVED** [review]

File: libs/viewer/src/Settings.cpp [hunk 1]
```diff
@@ -1351,6 +1353,7 @@ static std::ostream& operator<<(std::ostream& out, const ScreenSpaceReflectionsO
                << "\"thickness\": " << in.thickness << ",\n"
                << "\"bias\": " << in.bias << ",\n"
                << "\"maxDistance\": " << in.maxDistance << ",\n"
+               << "\"stride\": " << in.stride << ",\n"
```
Comment (prideout) [inline libs/viewer/src/Settings.cpp:1356]: I think this has the dreaded trailing comma of death 😱

**Review by prideout — CHANGES_REQUESTED** [review]
Trailing comma, otherwise looks great.

**Comment by lucasleandro1805** [comment]
@bejado please add reflection bindings in MaterialBuilder (Java/WEBGL).
We cannot change from default to screen-space reflection mode in MaterialBuilder using java.

**Pull request merged**